plugin-gradle/README.md

```diff
@@ -850,7 +852,32 @@ spotless {
 }
 ```
 
-<a name="applying-prettier-to-javascript--flow--typescript--css--scss--less--jsx--graphql--yaml--etc"></a>
+## Gherkin
+
+- `com.diffplug.gradle.spotless.GherkinExtension` [javadoc](https://javadoc.io/doc/com.diffplug.spotless/spotless-plugin-gradle/6.17.0/com/diffplug/gradle/spotless/GherkinExtension.html), [code](https://github.com/diffplug/spotless/blob/main/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/GherkinExtension.java)
+
+```gradle
+spotless {
+  gherkin {
+    target 'src/**/*.feature' // you have to set the target manually
+    simple() // has its own section below
+  }
+}
+```
+
+### simple
+
+Uses a Gherkin pretty-printer that optionally allows configuring the number of spaces that are used to pretty print objects:
```

Comment:
As per other formatters (e.g. [scalafmt](https://github.com/diffplug/spotless/tree/main/plugin-gradle#scalafmt)) link to

- https://github.com/cucumber/gherkin-utils
- https://github.com/cucumber/gherkin-utils/blob/main/CHANGELOG.md